packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -614,7 +614,13 @@ class HorizontalDragGestureRecognizer extends DragGestureRecognizer {
 ///    some time has passed.
 class PanGestureRecognizer extends DragGestureRecognizer {
   /// Create a gesture recognizer for tracking movement on a plane.
-  PanGestureRecognizer({ Object? debugOwner }) : super(debugOwner: debugOwner);
+  PanGestureRecognizer({
+    Object? debugOwner,
+    PointerDeviceKind? kind,
+  }) : super(
+         debugOwner: debugOwner,
+         kind: kind,
```

Comment:
`kind` is deprecated I think?

Reply:
Ah I missed that, thanks.  I'll do the same thing but with `supportedDevices` as the deprecation message suggests.